css/dashboard.css

```diff
@@ -50,8 +59,26 @@
   opacity: 1;
 }
 
+.wallpaper-swatches {
+  flex-flow: row nowrap;
```

Comment:
I would prefer if we could avoid shorthands for flex properties. I'm not familiar enough with them yet.